.github/workflows/build_packages.yaml

```diff
@@ -7,6 +7,9 @@ concurrency:
 on:
   schedule:
     - cron:  '0 */6 * * *'
+  push:
+    branch:
+      - 'ci/*'
```

Comment:
I want to keep it because it is convenient to test on my fork emqx repo.

Also nowadays it is not suggested to push the private branch to this emqx/emqx repo, so mostly it will not add load to the runners of this repo.

If you are worried about accidentally pushing the private built image to the docker hub, we have some guards on these jobs.